### ISSUE TITLE
Quiz Naive_Bayes

### ISSUE THREAD

**Issue by rosanalin** [rosanalin/Udacity_MchL#2]
def classify(features_train, labels_train):   
    ### import the sklearn module for GaussianNB
    from sklearn.naive_bayes import GaussianNB
    
    ### create classifier
    clf = GaussianNB()
    
    ### fit the classifier on the training features and labels
    clf.fit(features_train, labels_train)
    
    ### return the fit classifier
    pred = clf.predict(features_test)
    
    ### your code goes here!
